otter/test/convergence/test_gathering.py

```diff
@@ -127,23 +137,25 @@ def test_get_all_with_link_to_next_page(self):
         svcreq = resolve_retry_stubs(get_all_server_details(batch_size=10))
         fake_response = object()
         body = {'servers': servers[:10],
-                'servers_links': [{'href': 'nexturl1', 'rel': 'next'}]}
-        result = resolve_svcreq(svcreq, (fake_response, body), *self.req)
+                'servers_links': [{'href': 'url?limit=10&marker=9',
```

Comment:
I know it's not necessary, but I would feel better if this href had a full URL in it, like "https://example.com/url?..." to ensure the code is really ignoring everything in the URL before the query.